lib/actions/internal.coffee

```diff
@@ -59,7 +59,8 @@ exports.sudo =
 	signature: 'internal sudo <command>'
 	description: 'execute arbitrary commands in a privileged subprocess'
 	help: '''
-		Don't use this command directly!
+		WARNING: THIS INTERNAL COMMAND IS DEPRECATED and will be removed in the next
+		major version release of the CLI.
```

Comment:
* The only use I had found of `"internal sudo"` was removed in this PR, so I even thought of deleting the command straight away... But technically it would be a breaking change, hence opting for the warning instead. No hurry anyway.
* I've failed to see the advantage `"internal sudo"` over simply calling a function like `helpers.sudo(cmd)` with the desired command (CLI command or otherwise) as argument. Especially given that the implementation of internal sudo simply does `spawn('sudo')` or `windosu.exe()`, which is what `helpers.sudo()` does. The privilege elevation happens on the child process only.

Reply:
I think we could probably delete it, a major version change talks about the public facing interface, this was never designed to be public facing (it's also super weird that it existed in the first place...) I'd say remove.

Reply:
All right then... I've added a commit that deletes `internal sudo`.